### DISCUSSION TITLE
2.3.40 ISO image boots to 'grub'

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#3931] [Unsupported Versions] [answered]
Hello,

I am working on upgrading my 16.04x64 servers to 2.3.40 by doing a clean install.

I downloaded and verified the 2.3.40 ISO, created the bootable USB using rufus (tried balenaEtcher as well), and attempt to boot from it.

I get a 'grub' kernel and can't continue past it. Any advice?

**Comment by TOoSmOotH** [comment]
I have had better luck using Etcher

**Reply by glwallum** [reply]
Etcher didn’t even let me boot to it, the device was not listed in the boot menu. Do you have any ideas for that?

**Comment by Garigis** [comment, marked as answer]
You've probably figured something out by now, but if you haven't, I just got it to work by typing exit, then it loaded the Security Onion install page I was expecting in the first place.

It took much longer than it should have to figure that out. I was trying to figure out how to get the kernal to load, then was about to try using Etcher. I typed exit, then was going to shut down the machine when the installer page loaded.
